README.md

```diff
@@ -1124,181 +1124,71 @@ await descopeClient.management.audit.createEvent({
 });
 ```
 
-### Manage Authz
+### Manage FGA (Fine-grained Authorization)
 
-Descope support full relation based access control (ReBAC) using a zanzibar like schema and operations.
```

Comment:
now that we remove the authz from docs, is it deprecated? 

maybe worth add it as such (at least in @deprecated in docsjs)

Reply:
We will deprecate later